packages/app-desktop/services/plugins/UserWebviewDialog.tsx

```diff
@@ -113,7 +116,7 @@ export default function UserWebviewDialog(props: Props) {
 						viewId={props.viewId}
 						themeId={props.themeId}
 						borderBottom={false}
-						fitToContent={true}
+						fitToContent={props.fitToContent}
```

Comment:
Since you now moved the prop to the Dialog component, do we still need it here?

Reply:
Yes because I still want the Userwebview to have 100% width and 100% height.

Reply:
Something's not right with having this property on two nested components. Could you explain how it work when fitToContent is true, and when it's false? What's the dimensions of Dialog, and what's the dimensions of UserWebview?

Reply:
## UserWebview

When `fitToContent` is true, it'll be sized according to the content thus having exact pixel values. When it's false, it'll have 100% width and 100% height, filling it's parent completely.

We want this to be false so that the Webview grows to fill to the parent Dialog div.

## Dialog
When `fitToContent` is true, we'll leave the dimensions for this as `auto` in order to make it grow with the Webview. If it's false, we want to make this Dialog `90vw` wide `80vh` high. Ofcourse the Webview will always fill this 100%.

I did think about having the Dialog always at 90vh and 80vh but this caused issues like this: 
![image](https://user-images.githubusercontent.com/8184424/129396905-60421538-55e7-4aa5-ab80-2ff44e59394f.png)

The `iframe` has exact pixel dimensions but the Dialog div is way larger. _(it must be auto when fitToContent is on.)_

Reply:
So do you see the problem, and do you have any suggestion on how to fix it?

Reply:
I'm not sure where is the problem no?

Reply:
In short, To achieve full screen idea, we want the Dialog to have `90vw and 80vh` and the Webview must have 100% dimensions in order to fill this Dialog parent. Maybe there's a better way then what I did ?

Reply:
I couldn't quite see from the diff but your change makes sense actually. Thanks for clarifying, and let's merge then.